File: C#/8kyu/countingSheep.cs
// https://www.codewars.com/kata/54edbc7200b811e956000556/csharp

using System;
using System.Linq;

public static class Kata
{
    public static int CountSheeps(bool[] sheeps)
    {
        return sheeps.Where(e => e == true).ToArray().Length;
    }
}

File: JS/8kyu/countingSheep.js
// https://www.codewars.com/kata/54edbc7200b811e956000556/csharp

const arr = [true, true, true, false,
    true, true, true, true,
    true, false, true, false,
    true, false, false, true,
    true, true, true, true,
    false, false, true, true]

function countSheeps(arrayOfSheep) {
    return arrayOfSheep.reduce((r, e) => {
        if (e === true)
            r += 1
        return r
    }, 0)
}

console.log(countSheeps(arr))

// // Better Solution
function countSheeps(arrayOfSheeps) {
    return arrayOfSheeps.filter(Boolean).length;
}